cmd/kube-apiserver/app/server.go

```diff
@@ -244,15 +244,13 @@ func CreateNodeDialer(s completedServerRunOptions) (tunneler.Tunneler, *http.Tra
 		if s.KubeletConfig.Port == 0 {
 			return nil, nil, fmt.Errorf("must enable kubelet port if proxy ssh-tunneling is specified")
 		}
-		if s.KubeletConfig.ReadOnlyPort == 0 {
-			return nil, nil, fmt.Errorf("must enable kubelet readonly port if proxy ssh-tunneling is specified")
-		}
+		healthzPort := 10248
```

Comment:
hard-coding this will fail if the kubelet was started with an alternate `--healthz-port`